lib/commands/preload.ts

```diff
@@ -115,6 +115,7 @@ The file name must end with '.crt' and must not be already contained in the prel
 /etc/ssl/certs folder.
 Can be repeated to add multiple certificates.\
 `,
+			multiple: true,
```

Comment:
I think the `--add-certificate` option got broken in the conversion to oclif. `preload --help` and the forums specify that it _"can be repeated to add multiple certificates,"_ and presumably this worked with Capitano, but oclif requires `multiple: true` to be specified.